egs/sprakbanken/s5/local/chain/run_tdnn.sh

```diff
@@ -0,0 +1 @@
+tuning/run_tdnn_1b.sh
```

Comment:
It doesn't really matter, but best to use #!/bin/bash at the top here. Same for `run_tdnn_lstm.sh`